go.mod

```diff
@@ -7,34 +7,35 @@ require (
 	github.com/golangci/golangci-lint v1.28.3
 	github.com/google/addlicense v0.0.0-20200622132530-df58acafd6d5
 	github.com/jstemmer/go-junit-report v0.9.1
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.0.0-00010101000000-000000000000
```

Comment:
Were these deviations from https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/487/files#diff-37aff102a57d3d7b797f152915a6dc16 intentional?

Reply:
@bogdandrutu do you know why these were all changed ^

Reply:
Yes, these were intentional. Typically we just have the version be `v0.0.0` for modules in this repo. Not sure why it was updated. But @bogdandrutu 's changes are as desired, I'll leave it unchanged in this diff.